src/pages/InflectPage/components/Pali1AutoComplete/Pali1AutoComplete.tsx

```diff
@@ -23,7 +25,11 @@ export const Pali1AutoComplete = ({ db, initialValue, onChangePali1 }: Pali1Auto
 
     const loadOptions = () => {
       const results = db.exec(`SELECT pāli1 FROM '_stems' WHERE pāli1 like '${pali1}%' order by pāli1 asc`)
-      const pali1s = (results[0]?.values || []).flatMap((x: any[]) => x)
+      const pali1s = (results[0]?.values || [])
+        .flatMap((x: any[]) => x)
+        .sort((s1: string, s2: string) =>
+          PLS.stringCompare(PSC.convertAny(s1, PSC.Script.RO), PSC.convertAny(s2, PSC.Script.RO)),

```

Comment:
db will always have RO - this is as per the convention established in the vision document.